src/pages/ExplorePage.tsx

```diff
@@ -1,10 +1,40 @@
-import { Map, Map2 } from '@/organisms/Map';
+import { MapContainer } from '@/atoms/MapAtoms';
+import { Map } from '@/organisms/Map';
+import { Theme } from '@/theme/Theme';
+import MattapanZoningSubdistrictsData from '@/data/MattapanZoningSubdistricts';
+
+const paintGenerator = () => {
+	const paintArray: (string|number|string[])[] = [...Array(54).keys()];
+	const arrayWithColor = paintArray.map(x => [x, generateColor()])
+	const flattened = arrayWithColor.flat();
+	flattened.unshift('match', ['get', 'FID']);
+	flattened.push('blue');
+
+	return flattened
+}
+
+const generateColor = () => '#' + (Math.random() * 0xFFFFFF << 0).toString(16).padStart(6, '0');
+
+export const dataLayer = {
+  id: 'data',
+  type: 'fill',
+  paint: {
+    'fill-color': paintGenerator(),
+    'fill-opacity': [
+      'case',
+      ['boolean', ['feature-state', 'hover'], false],
+      .8,
+      0.5
+  ]
+  }
+};
+
+const mapData= MattapanZoningSubdistrictsData()
```

Comment:
Not sure if data and dataLayer should just be in this page component or passed in as a prop in the parent component (App.tsx)

Also I can't remember something something the reason to use the router to manipulate the data...? something like that, does anyone recall?